client/state/login/actions.js

```diff
@@ -37,6 +42,7 @@ const loginErrorMessages = {
 		'Please double check your information and try again.' ),
 	account_unactivated: translate( "This account hasn't been activated yet — check your email for a message from " +
 		"WordPress.com and click the activation link. You'll be able to log in after that." ),
+	sms_code_throttled: translate( 'You can only request a code via SMS once per minute. Please wait and try again.' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'You can only request a recovery code via SMS once per minute. Please wait and try again.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).